images/chaos-daemon/Dockerfile

```diff
@@ -14,9 +14,12 @@ ENV RUST_BACKTRACE 1
 
 RUN curl -L https://github.com/chaos-mesh/toda/releases/download/v0.2.0/toda-linux-amd64.tar.gz -o /usr/local/bin/toda.tar.gz
 RUN curl -L https://github.com/chaos-mesh/nsexec/releases/download/v0.1.5/nsexec-linux-amd64.tar.gz -o /usr/local/bin/nsexec.tar.gz
+RUN curl -L https://github.com/chaos-mesh/rs-tproxy/releases/download/v0.1.1/tproxy-linux-amd64.tar.gz -o /usr/local/bin/tproxy.tar.gz
```

Comment:
Now, `rs-tproxy` is another barrier in front of the the ARM chaos mesh 😢

Reply:
We can release an ARM binary as needed.